src/sonic-yang-models/yang-models/sonic-sflow.yang

```diff
@@ -0,0 +1,122 @@
+module sonic-sflow{
+
+    namespace "http://github.com/Azure/sonic-sflow";
+    prefix sflow;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-types {
+        prefix stypes;
+    }
+    import sonic-port {
+        prefix prt;
+    }
+    import sonic-vlan {
+        prefix svlan;
+    }
+    /*
+    import sonic-mgmt-port {
+        prefix smt;
+    }
+    */
+
+    description "SFLOW yang Module for SONiC OS";
+
+    revision 2021-04-26 {
+        description "First Revision";
+    }
+
+    container sonic-sflow {
+
+        container SFLOW_COLLECTOR {
+            list SFLOW_COLLECTOR_LIST {
+                max-elements 2;
+                key "name";
+
+                leaf name {
+                    type string {
+                        length 1..64;
+                    }
+                }
+
+                leaf collector_ip {
+                    mandatory true;
+                    type inet:ip-address;
+                }
+
+
+                leaf collector_port {
+                    type inet:port-number;
+                    default 6343;
+                }
+
+            } /* end of list SFLOW_COLLECTOR_LIST */
+        } /* end of container SFLOW_COLLECTOR */
+
+        container SFLOW_SESSION {
+            list SFLOW_SESSION_LIST {
+                key "port";
+
+                leaf port {
+                    type leafref {
+                        path "/prt:sonic-port/prt:PORT/prt:PORT_LIST/prt:name";
+                    }
+                }
+
+                leaf admin_state {
+                    type stypes:admin_status;
```

Comment:
Default admin-status?

Reply:
Default admin status is "up". 
https://github.com/Azure/sonic-swss/blob/master/cfgmgr/sflowmgr.cpp#L279

I've addressed the comment.